src/docs/sdk/performance/span-operations.mdx

```diff
@@ -0,0 +1,141 @@
+| ember    |              | Spans related to [EmberJS](https://emberjs.com/) |
+
+## Web Server
+
+Web server related spans should aim to follow OpenTelemetry's [HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md) and [RPC](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/rpc.md) semantic conventions when possible.
+
+| Category  | Usage           | Description                                                        |
+| --------- | --------------- | ------------------------------------------------------------------ |
+| http      |                 | Spans related to http operations                                   |
+|           | http.client     |                                                                    |
+|           | http.server     |                                                                    |
+| rpc       |                 | Spans related to remote procedure calls (RPC)                      |
+| grpc      |                 | Usage of the [gRPC framework](https://grpc.io/)                    |
+| template  |                 |                                                                    |
+|           | template.init   |                                                                    |
+|           | template.parse  |                                                                    |
+|           | template.render |                                                                    |
+| render    |                 | Rendering of a view                                                |
+| serialize |                 | Serialization of data                                              |
+| console   |                 | Accessing web servers through the command line (ex. Rails console) |
+
+## Web Frameworks
+
+| Category | Usage             | Description |
+| -------- | ----------------- | ----------- |
+| django   |                   |             |
+|          | django.middleware |             |
+|          | django.view       |             |
+| express  |                   |             |
+| rails    |                   |             |
+| rack     |                   |             |
+
+## Database
+
+Databased related spans are expected to follow OpenTelemetry's [Database](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md) semantic conventions when possible.
+
+| Category | Usage            | Description                |
+| -------- | ---------------- | -------------------------- |
+| db       |                  | An operation on a database |
+|          | db.connection    |                            |
+|          | db.transaction   |                            |
+|          | db.sql.query     |                            |
```

Comment:
Still a bit unclear what's the diff between `db.sql.query` and `db.query`? Should we just get rid of `db.sql.query`?

Reply:
According to the outlined spec `sql` is an identifier for the `db` category. I'm of the opinion that both are fine to exist - `db.query` is just the more general one. 

The key takeaway here is that since both use the same category, `db`, they will be classified the same way in the product. Everything after the `db` prefix just provides additional context, SDK developers should feel free to choose whatever they think fits the best (as long as the category stays consistent).

Open to discussing further if you feel that we should take a more opinionated approach.

Reply:
Ah alright, I thought it will affect the operation breakdown rendering in the UI, but if it's just there for context, I'm good with it, thanks